source/_includes/integrations/option_flow.md

```diff
@@ -7,4 +7,4 @@ Options for {{ name }} can be set via the user interface, by taking the followin
 - Browse to your Home Assistant instance.
 - Go to **{% my integrations title="Settings > Devices & Services" %}**.
 - If multiple instances of {{ name }} are configured, choose the instance you want to configure.
-- Select the integration, then select **Configure**.
+- Select the integration, then select **Add**.
```

Comment:
This should keep `**Configure**` since it is the options flow, not the config flow

Reply:
Oh, thank you for the catch!